README.md

```diff
@@ -81,6 +91,41 @@ $ rpc-client/build/install/rpc-client/bin/rpc-client issue -u localhost:10011 -n
 
 The client will discover all the participants in the network and prompt you for a node as the counterprise (borrower), make sure to pick the other node you created in the network, not the node you are connected to or the notary.
 
+### Account Utilities
+
+#### Create new account
+To create a new account with name `partyA` on a node:
+```
+$ rpc-client/build/install/rpc-client/bin/rpc-client account-utils --create-account --account-name partyA -u localhost:10011 -n user1 -p test
+```
+#### Generate a key pair for a given account
+To create a new key pair for a given account on a node:
```

Comment:
what happens if an account was created but didn't have a key pair? would be useful to add some explanation here why key pairs are not created together with the account.

if an account is not useable until a key pair is generated for it, then the key pair generation should be done in the `--create-account` step

Reply:
can multiple key pairs be generated for the same account, or newly generated one will replace the existing pair?

Reply:
A single account can have as many keys as possible. A table exist that has a mapping between accounts and public key hashes. So In our example even if a key pair is not explicitly generated, the iou issue works because a new key pair is generated when the account is used.  Usually For any transaction a new key is used (old key can also be used, not recommended). In above example a new key pair is used for any new transactions. While this utility for creating a new key pair for an account but it is not used in our example, but future examples can use it.  No both old and new key pair exist.